### DISCUSSION TITLE
Multilingual fields and their output via the GraphQL API

### DISCUSSION THREAD

**Discussion by paragliding-nsk** [torchbox/wagtail-grapple#373] [Q&A]
Hello, please tell me what tools are there for using multilingual fields with subsequent output via the GraphQL API?
In my project I have Wagtail + Grapple installed. There is wagtail documentation about "Internationalisation", here one can found something about API and models:
https://docs.wagtail.org/en/stable/advanced_topics/i18n.html#api-filters-for-headless-sites
But I just can’t find how to fully implement multilingual model fields and their output via the GraphQL API.

**Comment by zerolab** [comment]
Hey @paragliding-nsk 

Can you give https://github.com/torchbox/wagtail-grapple/pull/217 a try?

You would need to install it with

`pip install git+https://github.com/torchbox/wagtail-grapple.git@feature/pages-language-filter`

The PR adds pages filtering by locale as per https://docs.wagtail.org/en/stable/advanced_topics/api/v2/usage.html#filtering-pages-by-locale

**Reply by paragliding-nsk** [reply]
Thank you, it's interesting solution, but, as seems, for REST API only. I'm afraid it is not suitable for GraphQL API or may be I didn't found in documentation how to apply to GraphQl API.
